common/production_methods/02_agro.txt

```diff
@@ -0,0 +1,598 @@
+}
+
+pm_potatoes = {
+	texture = "gfx/interface/icons/production_method_icons/potatoes.dds"
+	
+	building_modifiers = {
+		workforce_scaled = {
+			building_output_grain_add = -20
+			building_output_liquor_add = 15
+		}
+	}
+}
+
+pm_apple_orchards = {
+	texture = "gfx/interface/icons/production_method_icons/orchards.dds"
+	
+	building_modifiers = {
+		workforce_scaled = {
+			building_output_grain_add = -20
+			building_output_fruit_add = 10
+			building_output_sugar_add = 5
+		}
+	}
+}
+
+pm_tools_disabled = {
+	texture = "gfx/interface/icons/production_method_icons/no_tool_use.dds"
+}
+
+pm_tools = {
+	texture = "gfx/interface/icons/production_method_icons/harvesting_tools.dds"
+	
+	building_modifiers = {
+		workforce_scaled = {
+			# input goods
+			building_input_tools_add = 1
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = -500
+		}
+	}
+}
+
+pm_steam_threshers = {
+	texture = "gfx/interface/icons/production_method_icons/steam_powered_threshers.dds"
+
+	pollution_generation = 10
+
+	unlocking_technologies = {
+		threshing_machine
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			# input goods													
+			building_input_tools_add = 2	
+			building_input_coal_add = 4
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = -1500
+		}
+	}
+}
+
+pm_tractors = {
+	texture = "gfx/interface/icons/production_method_icons/tractors.dds"
+
+	pollution_generation = 15
+
+	unlocking_technologies = {
+		mechanized_farming
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			# input goods								
+			building_input_coal_add = 5 
+			building_input_engines_add = 2
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = -2500
+		}
+	}
+}
+
+pm_compression_ignition_tractors = {
+	texture = "gfx/interface/icons/production_method_icons/tractors.dds"
+
+	pollution_generation = 20
+
+	unlocking_technologies = {
+		compression_ignition
+	}
+	building_modifiers = {
+		workforce_scaled = {
+			# input goods								
+			building_input_oil_add = 3
+			building_input_engines_add = 2
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = -3000
+		}
+	}
+}
+
+pm_privately_owned = {
+	texture = "gfx/interface/icons/production_method_icons/privately_owned.dds"
+
+	unlocking_laws = {
+		law_serfdom
+		law_tenant_farmers
+	}
+
+	building_modifiers = {
+		level_scaled = {
+			building_employment_aristocrats_add = 100
+		}
+		unscaled = {
+			building_aristocrats_shares_add = 10 
+			building_clergymen_shares_add = 2
+			building_farmers_shares_add = 0.25
+		}
+	}
+}
+
+pm_publicly_traded = {
+	texture = "gfx/interface/icons/production_method_icons/publicly_traded.dds"
+
+	unlocking_laws = {
+		law_commercialized_agriculture
+	}
+
+	building_modifiers = {
+		level_scaled = {
+			building_employment_aristocrats_add = 100
+			building_employment_capitalists_add = 50
+		}
+		unscaled = {
+			building_aristocrats_shares_add = 10 
+			building_capitalists_shares_add = 15 
```

Comment:
This may even not be significant enough, this is because it is (n_employees * shares) / total_shares = % of profit. This means that in a farm the aristocrats still get more % of the profits since they have 100 people while the capitalists have 50 people.

Reply:
Should we be changing how many capitalists there are too then, or just give them more shares?

Reply:
Either rebalance for a 75/75 employee split. Or make the shares at least double + 5 (25) to represent the change.